src/configs/firebase.js

```diff
@@ -0,0 +1,24 @@
+import {
+    API_KEY,
+    AUTH_DOMAIN,
+    DATABASE_URL,
+    PROJECT_ID,
+    STORAGE_BUCKET,
+    MESSAGING_SENDER_ID,
+    APP_ID
+  } from 'react-native-dotenv';
+  import * as firebase from 'firebase';
+  
+  var firebaseConfig = {
+    apiKey: API_KEY,
+    authDomain: AUTH_DOMAIN,
+    databaseURL: DATABASE_URL,
+    projectId: PROJECT_ID,
+    storageBucket: STORAGE_BUCKET,
+    messagingSenderId: MESSAGING_SENDER_ID,
+    appId: APP_ID,
+  };
+  
+  firebase.initializeApp(firebaseConfig);
+  
```

Comment:
Please make sure you complete all the points mentioned Issue #5, point 6 in the issue is not complete. Let me know if you have added it someplace else :smile: